src/desktop/test/models/article.test.coffee

```diff
@@ -326,6 +326,19 @@ describe "Article", ->
         layout: 'series'
       @article.hasAMP().should.be.false()
 
+    it 'returns false if article is not featured', ->
+      @article.set
+        sections: [ type: 'text' ]
+        featured: false
+      @article.hasAMP().should.be.false()
+
+    it 'returns false if article is a series', ->
```

Comment:
We should add a test here for `video` layout as well to make sure all our bases are covered.